models/monai_prostate158/utils/Prostate158Runner.py

```diff
@@ -0,0 +1,85 @@
+"""
+-------------------------------------------------
+MHub - MONAI Prostate158 Runner
+-------------------------------------------------
+
+-------------------------------------------------
+Author: Cosmin Ciausu
+Email:  cciausu97@gmail.com
+-------------------------------------------------
+"""
+# TODO: support multi-i/o and batch processing on multiple instances
+
+from typing import List, Optional
+import os, subprocess, shutil, glob, sys
+import SimpleITK as sitk, numpy as np
+from mhubio.core import Module, Instance, InstanceData, DataType, FileType, IO
+from mhubio.modules.runner.ModelRunner import ModelRunner
+import json
+
+@IO.Config('apply_center_crop', bool, False, the='flag to apply center cropping to input_image')
```

Comment:
Do we have explained (e.g. on the model card) when and why you want to apply center crop? I cannot find any notion when to disable this setting in the intended use. If the flag isn't actually optional we shouldn't expose it. If it's optional, we should give a hint of twhen to disable it ;)

Reply:
center_crop should be applied by default, since it is recommended by prostate158 for both training and inference.

I implemented here their center_crop strategy, enabled by default, but the user can disable it by passing center_crop=False in the .yml file.

Reply:
> center_crop should be applied by default, since it is recommended by prostate158 for both training and inference.

But the default value is `False`;)

Pls change to `@IO.Config('apply_center_crop', bool, True, the='flag to apply center cropping to input_image')`